docs/codeql/codeql-workshops/cpp/codeql-dataflow-sql-injection/qlpack.yml

```diff
@@ -0,0 +1,3 @@
+name: cpp-sql-injection
```

Comment:
It would be a good idea to give a common naming convention to the sample packs here, and make it obvious they are examples in case users copy them or accidentally publish them. For the starter repo we use `getting-started/<name>` since that cannot clash with a GitHub org name.

Reply:
gave them all a prefix, "codeql-workshop/<name>"